src/Error.php

```diff
@@ -97,13 +89,9 @@ public function getCode()
      * @param  string $message
      * @return self
      */
-    public function setMessage($message)
+    public function setMessage(string $message) : self
```

Comment:
We tend to drop fluent interfaces. It might be out of scope of that PR, but something what should be done noted and done for the next major.

Reply:
Agreed. I will open an issue as reminder and provide changes separately.